payment/views.py

```diff
@@ -2,44 +2,46 @@
 
 from rest_framework import generics, permissions
 
-from payment.models import Payment
+from borrowing.models import Borrowing
 from payment.serializers import (
-    PaymentSerializer,
     PaymentDetailSerializer,
-    PaymentCreateSerializer,
     PaymentListSerializer,
 )
 
 from .models import Payment
 
 
-class PaymentViewSet(
-    generics.CreateAPIView,
+class PaymentList(
     generics.ListAPIView,
-    generics.RetrieveAPIView,
     generics.GenericAPIView
 ):
+    queryset = Payment.objects.all()
```

Comment:
Maybe, we need to use select related for this, `cause this field "borrowing_id" is a foreign key?